lib/pages/full_post.dart

```diff
@@ -0,0 +1,99 @@
+import 'package:esys_flutter_share/esys_flutter_share.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_hooks/flutter_hooks.dart';
+import 'package:lemmy_api_client/src/models/post.dart';
+
+import '../widgets/comment_section.dart';
+import '../widgets/post.dart';
+
+class FullPostPage extends HookWidget {
```

Comment:
Should all pages have a `Page` suffix? If so, it has to be added to other pages

Reply:
yes, `Page` or `Tab` suffix